examples/compute/CMakeLists.txt

```diff
@@ -48,4 +48,4 @@ find_library(WGPU_LIBRARY wgpu_native
 
 target_include_directories(${TARGET_NAME} PUBLIC $ENV{GLFW3_INCLUDE_DIR})
 
-target_link_libraries(${TARGET_NAME} glfw ${WGPU_LIBRARY} ${OS_LIBRARIES})
+target_link_libraries(${TARGET_NAME} glfw ${WGPU_LIBRARY} ${OS_LIBRARIES})
```

Comment:
no need to delete that last empty line :)

Reply:
Bad habits 😅